templates/engage/gsi-webinar-series.html

```diff
@@ -0,0 +1,237 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Building powerful customer solutions{% endblock %}
+
+{% block meta_description %}A webinar series for GSIs seizing new models for scalable, automated and repeatable deployments{% endblock %}
+
+{% block meta_image %}https://assets.ubuntu.com/v1/533d8f4a-social-gsi-webinar-series.png{% endblock meta_image %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1ZweTXyHO9kIU9aE8pw2_Ni_NoBShM_KseY5H2XtGueg/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<style>
+  .global-nav .global-nav__header-logo-anchor {
+    padding-left: 0 !important;
+  }
+</style>
+<section class="p-strip p-engage-banner--dark">
+  <div class="u-fixed-width navigation-logo-engage">
+    <a href="/">
+      {{
+        image(
+        url="https://assets.ubuntu.com/v1/f263d9c4-logo-ubuntu-white.svg",
+        alt="Ubuntu",
+        width="143",
+        height="32",
+        hi_def=True,
+        loading="auto",
+        ) | safe
+      }}
+    </a>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-7  u-vertically-center">
+      <div>
+        <h1>
+          Building powerful customer solutions
```

Comment:
The copy doc has "Building powerful customer solutions on Ubuntu"

Reply:
I removed that... I should have updated the copy doc